connectors/src/connectors/notion/temporal/workflows/index.ts

```diff
@@ -179,70 +177,32 @@ export async function notionSyncWorkflow({
   // wait for all child workflows to finish
   await Promise.all(promises);
 
-  // TODO: remove else block when incr-sync-crawl patch is removed
-  // Next phase is to call deprecatePatch("incr-sync-crawl")
-  // https://docs.temporal.io/develop/typescript/versioning#patching
-  if (patched("incr-sync-crawl")) {
```

Comment:
Everything in this file is basically a no-op. Taking the opportunity that we're changing the workflow version to remove the patch, since we're causing a restart anyway.